content/docs/command-reference/queue/kill.md

```diff
@@ -39,5 +39,5 @@ To kill all running experiments and also stop processing the queue, use
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-- `-f`, `--force` - Forcefully and immediately kill running experiment queue
-  tasks.
+- `-f`, `--force` - forcefully and immediately interrupt the task (not
+  graceful).
```

Comment:
Quick Q. `queue stop --kill` is also "forced" right? I.e. not graceful cc @dberenbaum

Reply:
Good point @jorgeorpinel. @karajan1001 What do you think?

Reply:
Feel free to make this an issue in the core repo if you agree (or I can, lmk). Thanks